bundles/org.openhab.binding.emotiva/README.md

```diff
@@ -0,0 +1,186 @@
+| general#input_[1-8]         | String (R)   | User assigned input names                                  |
+| general#selected_mode       | String (R)   | User selected mode for the main zone                       |
+| general#selected_movie_music | String (R)   | User selected movie or music mode for main zone            |
+| general#mode_ref_stereo     | String (R)   | Label for mode: Reference Stereo                           |
+| general#mode_stereo         | String (R)   | Label for mode: Stereo                                     |
+| general#mode_music          | String (R)   | Label for mode: Music                                      |
+| general#mode_movie          | String (R)   | Label for mode: Movie                                      |
+| general#mode_direct         | String (R)   | Label for mode: Direct                                     |
+| general#mode_dolby          | String (R)   | Label for mode: Dolby                                      |
+| general#mode_dts            | String (R)   | Label for mode: DTS                                        |
+| general#mode_all_stereo     | String (R)   | Label for mode: All Stereo                                 |
+| general#mode_auto           | String (R)   | Label for mode: Auto                                       |
+| general#mode_surround       | String (RW)  | Select audio mode (Auto, Stereo, Dolby, ...)               |
+| general#width               | Number (RW)  | Width Volume increment up/down (0.5 step)                  |
+| general#height              | Number (RW)  | Height Volume increment up/down (0.5 step)                 |
+| general#bar                 | String (R)   | Text displayed on front panel bar of device                |
+| menu_display_highlight      | String (R)   | Menu Panel Display: Value in focus                         |
+| menu_display_top_start      | String (R)   | Menu Panel Display: Top bar, start cell                    |
+| menu_display_top_center     | String (R)   | Menu Panel Display: Top bar, center cell                   |
+| menu_display_top_end        | String (R)   | Menu Panel Display: Top bar, end cell                      |
+| menu_display_middle_start   | String (R)   | Menu Panel Display: Middle bar, start cell                 |
+| menu_display_middle_center  | String (R)   | Menu Panel Display: Middle bar, center cell                |
+| menu_display_middle_end     | String (R)   | Menu Panel Display: Middle bar, end cell                   |
+| menu_display_bottom_start   | String (R)   | Menu Panel Display: Bottom bar, start cell                 |
+| menu_display_bottom_center  | String (R)   | Menu Panel Display: Bottom bar, center cell                |
+| menu_display_bottom_end     | String (R)   | Menu Panel Display: Bottom bar, end cell                   |
+
+(R) = read-only (no updates possible)
+(RW) = read-write
+
+## Full Example
+
+`.things` file:
+
+```perl
```

Comment:
```suggestion
```java
```

Reply:
I lifted this of one of the other bindings, if I change to java, IDEA atleast starts to give errors.

Reply:
In which binding did you find that for a .things definition? This was fixed recently for all bindings - see PR's linked to #13858.